.gitignore

```diff
@@ -15,3 +15,12 @@ src-gen/
 setup.log
 local.properties
 wurstpack_compiler.zip
+buildproject/
+compiled.j.txt
+myname/
+ptrtestproject/
```

Comment:
don't add your local test folders to gitignore

Reply:
I think these are all generated by the project itself - what do you mean "local test folders"?

Reply:
what makes u think
```
compiled.j.txt
myname/
ptrtestproject/
```

would be files generated by this project?!

Reply:
> would be files generated by this project?!

The project has tests, right? What kind of tests does it have? Wurst tests

In fact I _know_ these are generated by some gradle task, because otherwise I wouldn't have these files to ignore